issue_template.md

```diff
@@ -1,10 +1,33 @@
-<!-- BUGS: Please use this template. -->
-<!-- QUESTIONS: This is not a general support forum! Ask Qs at http://stackoverflow.com/questions/tagged/typescript -->
-<!-- SUGGESTIONS: See https://github.com/Microsoft/TypeScript-wiki/blob/master/Writing-Good-Design-Proposals.md -->
+<!-- 🚨 𝑺𝑻𝑶𝑷 🚨 𝑺𝑻𝑶𝑷 🚨 𝑺𝑻𝑶𝑷 🚨 -->

```

Comment:
As much as they should be, your italicized Unicode letters may not be visible for all  You should have a fallback for these. Maybe alternate 𝑺𝑻𝑶𝑷 and STOP for good measure?